apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/commons/BBCodingSystems.java

```diff
@@ -0,0 +1,48 @@
+package gov.cms.bfd.server.war.commons;
+
+public class BBCodingSystems {
```

Comment:
I'll keep beating this drum, but class-level documentation. (and on the inner classes as well). While this class seems relatively straightforward, having intention declared helps avoid classes like these from turning into a variable dumping ground later.